app/utilities/schema.py

```diff
@@ -80,10 +81,26 @@ def get_allowed_languages(schema_name, launch_language):
 
 
 def load_schema_from_metadata(metadata):
-    if metadata.get("survey_url"):
-        return load_schema_from_url(
-            metadata["survey_url"], metadata.get("language_code")
+    if survey_url := metadata.get("survey_url"):
+        # :TODO: Remove before production uses survey_url
+        # This is temporary and is only for development/integration purposes.
+        # This should not be used in production.
+
+        start = time.time()
+        schema = load_schema_from_url(survey_url, metadata.get("language_code"))
+        duration = time.time() - start
+
+        cache_info = (
+            load_schema_from_url.cache_info()  # pylint: disable=no-value-for-parameter
+        )
+        logger.info(
+            f"load_schema_from_url took {duration} seconds",
```

Comment:
You might want to format the output here to milliseconds or something, it's a bit hard to grok scientific notation:

`load_schema_from_url took 9.5367431640625e-07 seconds`